packages/api-logs/index.jsx

```diff
@@ -88,7 +88,7 @@ class Logs extends React.Component {
   }
 
   onSelect(event) {
-    this.changeGroup(event.target.value);
+    this.onGroupChange(event.target.value, event.target.options[event.target.selectedIndex].text);
```

Comment:
Can the group name be inferred from the value higher up the stack? Or does it make sense to always have to pass in the pair here?